WMCDuplicateRemover/StaticValues.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace WMCDuplicateRemover
+{
+    public static class StaticValues
+    {
+        public static readonly string WMCDuplicateRemoverApplicationDataFolder = $"{Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData)}\\WMCDuplicateRemover";
+    }
+}
```

Comment:
Is the point of this class just to house values that don't change?  So you'll just create an instance of this class and use it to reference various values?  (I see that it only has one now, but it seems like you plan to expand it)